core/src/main/java/org/nervos/ckb/exceptions/APIErrorException.java

```diff
@@ -0,0 +1,16 @@
+package org.nervos.ckb.exceptions;
+
+import java.io.IOException;
+
+/**
+ * Created by duanyytop on 2018-12-26.
+ * <p>
```

Comment:
Is this `<p>` IDE specific? It looks like an open HTML element?

Reply:
em..., IDE add automatically. I remove it